pom.xml

```diff
@@ -54,8 +55,8 @@
         <org.projectlombok.version>1.16.18</org.projectlombok.version>
 
         <azure.documentdb.version>1.15.2</azure.documentdb.version>
-        <azure.test.resourcegroup>gremlin-test-4-11</azure.test.resourcegroup>
-        <azure.test.dbname>springdocdbtestdb</azure.test.dbname>
+        <azure.test.resourcegroup>cosmosdb-test-5-28</azure.test.resourcegroup>
+        <azure.test.dbname>testdb-${maven.build.timestamp}</azure.test.dbname>
```

Comment:
Since the timestamp is only at the minute level, is it possible that two tests triggered in the same minute compete against each other?

Reply:
The build time may takes about 1 minutes include default UT. So that situation hardly happens in my opinion. If really occurs, just rerun the test is acceptable.

Reply:
If 2 PR triggered build and test simultaneously, one would definitely fail due to creation of the db with the same name. In my opinion, it's not a good practice.

Reply:
Maybe a random suffix will solve the problem

Reply:
But that rarely happens in our routine work. Even if you use milliseconds, you can not avoiding concurrent PR triggering, just low down the possibility indeed.

Reply:
Random can be solution for that, but indeed it is also the issue of possibility.